### PR TITLE
Added multi-day to the full-page booking form

### PR DESCRIPTION
**Pull request by sfedorenko**

### CODE REVIEW

File: src/fullPageBookingForm/Components/TimeslotCard/TimeslotCard.tsx [hunk 1]
```diff
@@ -1,36 +1,57 @@
...
   endsAt: Date;
   timezone: string;
+  formattedTimeslot: FormattedTimeslot;
   remainingSpots: number;
   minPrice: number;
   onSelect: () => void;
   moneyFormat: string;
   labels: Partial<AppDictionary>;
+  occurrenceId?: string;
+  occurrence?: {
```
Comment (shannon-coding) [inline src/fullPageBookingForm/Components/TimeslotCard/TimeslotCard.tsx:24]: Would  you want to break this into a separate type? Is it used elsewhere?

**Review by ckeyes88 — APPROVED** [review]
Looks good @sfedorenko. I second @shannon-coding question about making that typing reuseable.

Also, what's the compatibility of these changes with the current functionality. If this gets merged, do we need to wait until multi-day is released to release this or will it continue working with current experience versions?

File: src/fullPageBookingForm/Components/TimeslotCard/TimeslotCard.tsx (hunk 1 above)
Comment (sfedorenko) [reply src/fullPageBookingForm/Components/TimeslotCard/TimeslotCard.tsx:24]: I agree that it should be separated into a new type.
Irene and I will do this on Monday, as this change applies to all parts of our application (ExperiencesAppLibs, ExperiencesPlatform, ShopifyExpApp, and ExperiencesWidgets).

**Comment by sfedorenko** [comment]
> Looks good @sfedorenko. I second @shannon-coding question about making that typing reuseable.
> 
> Also, what's the compatibility of these changes with the current functionality. If this gets merged, do we need to wait until multi-day is released to release this or will it continue working with current experience versions?

We can merge these branches at any time. But let's do this after the "Occurrence" type is added.

**Pull request merged**